### ISSUE TITLE
Saved Searches

### ISSUE THREAD

**Issue by Philiphlop** [notable/notable#589]
### Feature description

Ability to save searches for as quick folders/tags/views.

This is done with other tools to see how it's implemented:
Outlook: Search folders
Evernote: Saves searches
Standard notes: [Smart tags](https://standardnotes.org/help/42/how-do-i-view-a-list-of-untagged-notes)

### Feature motivation

If I create a search for tag:Project/ABC tag:Status/Active before:2019-03-01 I'd like to save it as a Project Review shortcut somewhere. This saves me time from having to keep searching again and again, also having to remember the search string each time.

**Comment by fabiospampinato** [comment]
As of v1.6 you can now link to searches like this: `[](@search/Foo)`, you could use this syntax to make a list of saved searches.

This might not be ideal, but maybe it's not too bad either?

IMHO a more useful feature, at least when considering a single-user environment, would be to have a way to access the previous searches, there are multiple ways to implement this:

- Pressing up while focusing on the search field.
- Showing an autocomplete floating panel.
- Adding a custom command palette for this.

What do you guys think would be the best way to implement this?

**Comment by jakebrinkmann** [comment]
Maybe a section on the navigation pane ("All Notes", "Favorites", "Tags", _**"Searches"**_) ?

**Comment by bkonia** [comment]
I agree, we need a Searches section in the sidebar. A list of previous searches would not be particularly useful to me, because the list would quickly become cluttered with all kinds of junk; searches that were performed once and not needed again. The searches section should only display searches that you've specifically saved.
